phoenix/app/apiProducts/test/apiProducts/redis_cache_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule ApiProducts.RedisCacheTest do
+  use ApiProductsWeb.ConnCase, async: false
+
+  alias ApiProducts.Cache
+
+  setup_all do
+    %{
+      key: "some_key",
+      value: "some_value",
+      invalid_key: "invalid_key"
+    }
+  end
+
+  describe "Cache.set" do
+    test "save data in cache", %{key: key, value: value} do
+      assert Cache.set(key, value) == {:ok, "OK"}
```

Comment:
É bom verificar se o valor esta mesmo cacheado. Parecido com o que fez no teste `"get data in cache if key is valid"`